src/test/java/step_definitions/SalesForceMarketing.java

```diff
@@ -82,120 +82,120 @@ public void Select_Parent_Campaign() throws InterruptedException {
     }
 
 
-    @Then("user selects the edit button")
+    @Then("User selects the edit button")
     public void campaign_Edit() throws InterruptedException {
         CampaignUtil.editCampaignName(dataMap.get("CampaignName"), dataMap.get("EditCampaignName"));
         CampaignUtil.verificationMessage();
         KeywordUtil.takeScreenshotAndAttachInReport();
     }
 
-    @Then("user selects the Delete button")
+    @Then("User selects the Delete button")
     public void campaign_Delete() throws InterruptedException {
         CampaignUtil.deleteCampaign(dataMap.get("EditCampaignName"));
         CampaignUtil.verificationMessage();
         KeywordUtil.takeScreenshotAndAttachInReport();
     }
 
-    @Then("user selects the parent comapaign")
+    @Then("User selects the parent comapaign")
     public void select_ParentCampaign() throws InterruptedException {
         System.out.println(dataMap.get("ParentCampaign"));
         CampaignUtil.addParentCampaign(dataMap.get("CampaignName"), dataMap.get("ParentCampaign"));
     }
 
-    @And("user clicks on the Campaign")
+    @And("User clicks on the Campaign")
     public void click_Campaign() throws InterruptedException {
         CampaignUtil.clickCampaignName(dataMap.get("CampaignName"));
         KeywordUtil.delay(6000);
         KeywordUtil.takeScreenshotAndAttachInReport();
     }
 
-    @Then("user create contact in Campaign")
+    @Then("User create contact in Campaign")
     public void create_Contact() throws InterruptedException {
         CampaignUtil.addContactInCampaign(dataMap.get("Salutation"), dataMap.get("ContactLastName"), KeywordUtil.generateRandomString(4) + "@gmail.com", dataMap.get("Phone"));
         CampaignUtil.verificationMessage();
         KeywordUtil.takeScreenshotAndAttachInReport();
     }
 
-    @And("user Add the leads in campaign")
+    @And("User Add the leads in campaign")
     public void click_addLeads() throws InterruptedException {
         CampaignUtil.addLeads(dataMap.get("LeadName"));
 
     }
 
-    @Then("user clicks on the Contact tab")
+    @Then("User clicks on the Contact tab")
     public void click_contactTab() throws InterruptedException {
         CampaignUtil.clickContactTab();
 
     }
 
-    @Then("user selects the filter")
+    @Then("User selects the filter")
     public void selectFilter() throws InterruptedException {
```

Comment:
Sometimes you are using underscore for the function names and sometimes without it. Please use same naming convention for functions.